fsh/patient.fsh

```diff
@@ -53,3 +54,10 @@ Id: isolation
 Title: "Isolation"
 Description: "If the monitoree should be in the isolation workflow (omitting this extension defaults this value to false, leaving the monitoree in the exposure workflow."
 * value[x] only boolean
+
+// Full Assigned Jurisdiction Path
+Extension: FullAssignedJurisdictionPath
+Id: full-assigned-jurisdiction-path
+Title: "Full Assigned Jurisdiction Path"
+Description: "Represents the jurisdiction in which this Patient is enrolled."
```

Comment:
You could consider having this description use "monitoree" instead of "Patient", for sake of consistency with other descriptions.